lib/reports/ReportBaseCrash.js

```diff
@@ -51,9 +40,17 @@ class ReportBaseCrash extends ReportBase {
     return { collisions, collisionSummary };
   }
 
-  async generateLayoutHeader(features) {
-    const locations = await CentrelineDAO.byFeatures(features);
-    const info = getLocationsDescription(locations);
+  async generateLayoutHeader({ locationsSelection }) {
+    let info = getLocationsSelectionDescription(locationsSelection);
+    /*
+     * `pdfmake` only supports ANSI characters:
+     *
+     * https://pdfmake.github.io/docs/0.1/fonts/standard-14-fonts/
+     *
+     * To work around this, we replace the Unicode arrow used in corridor descriptions
+     * with a more ANSI-friendly equivalent here.
+     */
+    info = info.replace('\u2192', '-');
```

Comment:
we should also handle `\u2013` here, which is generated by `MidblockDescription` for midblocks with two detectable cross streets.